src/core/commit.cpp

```diff
@@ -176,17 +176,20 @@ class CommitComponentImpl : public ComponentBase, public WindowOptions {
             }
         } else if (ostreetui.GetViewMode() == ViewMode::COMMIT_DROP &&
                    ostreetui.GetModeHash() == hash) {
-			const auto& commitList = ostreetui.GetOstreeRepo().getCommitList();
-			auto commit = commitList.at(hash);
-			auto it = std::find_if(
-    		    commitList.begin(), commitList.end(),
-    		    [&](std::pair<std::string, cpplibostree::Commit> c) { return c.second.branch == commit.branch; });
-    		if (it != commitList.end() && commit.hash == it->second.hash) {
-    		    startDeletionWindow();
-    		} else {
-				ostreetui.notificationText = "Can't drop commit " + commit.hash.substr(0,8) + "... not last commit on branch " + commit.branch + " (which is " + it->second.hash.substr(0,8) + ")";
-				cancelSpecialWindow();
-			}
+            const auto& commitList = ostreetui.GetOstreeRepo().getCommitList();
+            auto commit = commitList.at(hash);
+            auto it = std::find_if(commitList.begin(), commitList.end(),
```

Comment:
```suggestion
            const auto commit = commitList.at(hash);
            const auto it = std::find_if(commitList.begin(), commitList.end(),
```

Explicit const all the things :)

Reply:
I'm not sure what went wrong here, but a lot of these change requests are outdated, without me even changing anything. They are not the most current version and are not present in the most up-to-date commit...